WireDomain/Sources/WireDomain/Storage/UpdateEventsUserDefaults/UpdateEventsUserDefaults.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public enum UpdateEventsUserDefaults: UserDefaultsProtocol {
```

Comment:
I wonder why using this as static property instead of wrapping this in a concrete object

Reply:
The purpose is to to provide a **scoped** UserDefaults API that is designed to be simple and intuitive to use for clients who consume it : from other packages, no need for initialization, you can call the static property directly and UserDefaults are scoped depending on their domain concern: for instance, we could have a `enum ConversationUserDefaults: UserDefaultsProtocol` or `enum FederationUserDefaults: UserDefaultsProtocols` etc that would also be exposed by WireDomain.